src/course/syllabus/pre-apprenticeship/projects/project.md

```diff
@@ -2,7 +2,14 @@ You'll be working on a solo project in your free time for the last 3 weeks. We e
 
 These projects are designed to help you revise the fundamentals you have learnt during the first 9 weeks of the course. We recommend you pick one that involves something you are uncomfortable with. For example if you struggled using fetch and promises then pick a project that will force you to practice that.
```

Comment:
This is still true but perhaps add a sentence on setting up a dev environment?

Reply:
I figured that would be obvious from the workshops + technical requirements just below